forte/ontology_specs/base_ontology.json

```diff
@@ -399,6 +399,21 @@
           "item_type": "int"
         }
       ]
+    },
+    {
+      "entry_name": "ft.onto.base_ontology.MRCAnswer",
+      "parent_entry": "forte.data.ontology.top.Annotation"
+    },
+    {
+      "entry_name": "ft.onto.base_ontology.MRCQuestion",
+      "parent_entry": "forte.data.ontology.top.Annotation",
+      "attributes": [
```

Comment:
Let's include "description" key here. Did you update the `base_ontology.py` docstring manually? We don't have to do so.